### ISSUE TITLE
Can't update category and article via php api

### ISSUE THREAD

**Issue by drcsystems78** [zendesk/zendesk_api_client_php#404]
Update section is working fine but Article and Category are not update, Help me what i am doing wrong? See below
`// for article
$client = $this->helper->getZendeskClient();
$article = $client->helpCenter->sections($postData["section"])->articles()->update($postData["id"],[
 'locale' => 'en-us',
 'title' => $postData["title"],
'author_id'=> $postData["author"],
'body'=> $postData["body"]
]);
// for category
$client = $this->helper->getZendeskClient();
$category = $client->helpCenter->categories()->update($postData["id"],[
'position' => 0,
'locale' => 'en-us',
'name' => $postData["title"],
]);`

**Comment by jmramos02** [comment]
can you verify if the `$postData['id']` variable has any valid value? snippet looks good to me

**Comment by drcsystems78** [comment]
Yes i verified it, `$postData["id"]` getting perfect value, It's id for category and article.

**Comment by jmramos02** [comment]
I'll try to replicate this on my local. will get back to you

**Comment by ecoologic** [comment]
Hi,

 thank you for your request,
are you still interested in a solution for this issue?

**Comment by scuben** [comment]
Hi,

No, thank you.

BR,
Patrick

--------------------------------------
Patrick Landolt
Geschäftsführer

ARTACK WebLab GmbH
Schulstrasse 37
8050 Zürich

T: +41 44 222 22 11
D: +41 44 222 22 16
W: https://www.artack.ch
--------------------------------------

www.mailXpert.ch
Professionelles E-Mail Marketing von ARTACK WebLab GmbH
